src/lib/core/ripple/ripple.spec.ts

```diff
@@ -7,17 +7,15 @@ import {
   MdRipple, MdRippleModule, MD_RIPPLE_GLOBAL_OPTIONS, RippleState, RippleGlobalOptions
 } from './index';
 
-/** Extracts the numeric value of a pixel size string like '123px'.  */
-const pxStringToFloat = (s: string) => {
-  return parseFloat(s.replace('px', ''));
-};
 
 describe('MdRipple', () => {
   let fixture: ComponentFixture<any>;
   let rippleTarget: HTMLElement;
-  let originalBodyMargin: string;
+  let originalBodyMargin: string | null;
   let viewportRuler: ViewportRuler;
 
+  /** Extracts the numeric value of a pixel size string like '123px'.  */
+  const pxStringToFloat = (s: string | null) => s ? parseFloat(s.replace('px', '')) : 0;
```

Comment:
I know it was already like this, but doesn't `parseFloat` already ignore trailing non-numeric characters already?